Gemfile

```diff
@@ -23,16 +23,16 @@ gem "json_api_client", ">= 1.21.1"
 gem "lograge", ">= 0.11.2"
 gem "logstash-event"
 gem "mail-notify"
-gem "omniauth", "~> 2.1"
-gem "omniauth-oauth2", "~> 1.8"
+gem "omniauth"
+gem "omniauth-oauth2"
 gem "omniauth-rails_csrf_protection"
 gem "pagy"
 gem "paper_trail", "~> 15.0"
 gem "pg", ">= 0.18", "< 2.0"
 gem "pg_search"
 gem "puma", "~> 6.4.0"
 gem "rack-attack"
-gem "rails", "~> 6.1.7"
+gem "rails"
```

Comment:
Wouldn't be better to specify minor version here?

Reply:
I am curious about this one. 

If we are using `dependabot` for gems upgrades, and the update is descriptive and small enough of the version that is being upgraded, Isn't it better to receive all the updates for Rails all upgrades once they are available? 

I would like to see an update from version `7.1.2` to `7.2` as soon as it happens because at the very end, we will have to do the update as frequently as possible to decrease risk.

Wouldn't it be better to just say: `gem "rails", "7"` ?

I won't fight much for it :), just curious about your thoughts. I noticed that ECF also fix the minor version

cc/ @peteryates

Reply:
I don't have any strong reservations or thoughts on version. so long as we're keeping on top of it and current or nearly-current 😅

I'd probably be inclined to go with `7` or `7.1` and let dependabot take care of the rest.

Reply:
I have finally gone with 

```ruby
gem "rails", "~> 7.0" 
```

This would allow updates from 7.0.x to 7.1.x, 7.2.x, etc., but not to 8.0.0